### ISSUE TITLE
Detailed information about audio and video streams

### ISSUE THREAD

**Issue by WanWizard** [OpenPLi/enigma2#971]
See https://forums.openpli.org/topic/51083-show-number-of-channels-for-dolby-digital/

**Comment by Dream1975** [comment]
Maybe some code from https://sourceforge.net/projects/mediainfo/ can be borrowed and integrated into enigma. The licence permits it

**Comment by nautilus7** [comment]
A similar request would be to show whether a tv channel is encoded in HDR (high dynamic range) or not. 

This can be determined by examining the HEVC bitstream and check if the PQ (perceptual quantizer) is present of not.

**Comment by athoik** [comment]
Hi, 

Reporting gamma curve was implemented on #1777. Drivers should send video event 17 and framerate can contain one of the possible gamma values (documented on the commit).

```
[eDVBVideo0] VIDEO_GET_EVENT SIZE_CHANGED 3840x2160 aspect 3
[eDVBVideo0] VIDEO_GET_EVENT FRAME_RATE_CHANGED 50000 fps
[eDVBVideo0] VIDEO_GET_EVENT PROGRESSIVE_CHANGED 1
[eDVBVideo0] VIDEO_GET_EVENT GAMMA_CHANGED 3
```

![1_0_1f_2_40b_1_c00000_0_0_0_20181018185555](https://user-images.githubusercontent.com/2682247/47171630-dbc7a880-d311-11e8-8164-29fa4d53c396.jpg)
